cfn-resources/project/cmd/resource/resource.go

```diff
@@ -75,6 +75,7 @@ func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler
 		Name:                      *currentModel.Name,
 		OrgID:                     *currentModel.OrgId,
 		WithDefaultAlertsSettings: currentModel.WithDefaultAlertsSettings,
+		RegionUsageRestrictions:   *currentModel.RegionUsageRestrictions,
```

Comment:
this new property is being dereferenced, is not set as a default or validated, not set as required, and not set in the required fields, so if the user does not provides it, is going to throw a null pointer

Reply:
how did I miss this.. :( updated now.